tvm/version_manager/infrastructure/version_manager_git_repository.py

```diff
@@ -148,6 +150,26 @@ def create_virtualenv(self, version: TutorVersion) -> None:
                        shell=True, check=True,
                        executable='/bin/bash')
 
+    def run_command_in_virtualenv(self, options: List):
+        try:
+            subprocess.run(f'source {self.TVM_PATH}/venv/bin/activate;'
+                       f'pip {" ".join(options)}',
+                       shell=True, check=True,
+                       executable='/bin/bash')
+        except subprocess.CalledProcessError as ex:
+            click.echo(click.style(
```

Comment:
I think that it is better to use an exception:
```python
raise Exception(f'Error running venv commands: {ex.output}')
```
then capture it and print it on cli.py